deployer/aws-lambda/content-origin-request/package.json

```diff
@@ -4,7 +4,8 @@
   "license": "MPL-2.0",
   "main": "index.js",
   "scripts": {
-    "make-package": "yarn install && zip -r -X function.zip . -i index.js 'node_modules/*'"
+    "make-package": "yarn install --production && zip -r -X function.zip . -i index.js 'node_modules/*'",
```

Comment:
This is an important change. If you `yarn install --production` it will not put any of the `devDependencies` in the `node_modules` which means we can add whatever to the `devDependencies` without bloating the `.zip` that gets uploaded.